src/github-handler/fork-handler.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Logger, Octokit} from '../types';
+
+interface ForkData {
+  forkOwner: string;
+  forkRepo: string;
+}
+
+/**
+ * Fork the GitHub owner's repository
+ * If fork already exists no new fork is created and no error occurs
```

Comment:
If no error is returned, what is returned?

Reply:
I have a @returns statement on line 29. I'll also add a commit to update this comment.

Reply:
I meant in the case of the fork already existing. Does the GitHub API `createFork` return the same data if the fork already exists?

Reply:
It returns a superset of the original data. I believe it also adds a `parent` and `source` object.

Reply:
Can you add to this note that it will return the existing fork?

Reply:
Just as an FYI what I originally said isn't true. It returns the exact same data each time (except for the `updated_at` field). It's just the GitHub docs don't include the parent/source object.